src/user/user.service.ts

```diff
@@ -70,15 +74,18 @@ export class UserService {
             },
             data:updatedData
         })
-        return updatedUser
+        const action = this.actionsService.addAction('Update',user.id,'User',updatedUser.id,updatedUser)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them